buttons.css

```diff
@@ -0,0 +1,24 @@
+.btn--border {
+    border: solid 1px var(--gray-2);
+    border-radius: 0.3em;
+    padding: 0.5em;
+}
+
+.btn {
+    font-size: inherit;
+    background-color: var(--white);
+    color: var(--primary-gray);
+}
+
+.btn--no-border {
+    border: 0;
+}
```

Comment:
When we have "boolean" states - like the border one which can exist or not, it's best to have a default version.

For example, here you could say that the `no-border` variant is default. Thus, move this rule inside `.btn` and delete the `btn--no-border` class.